.github/workflows/cluster_endtoend_mysql80.yml

```diff
@@ -34,33 +34,16 @@ jobs:
 
     - name: Get dependencies
       run: |
-
```

Comment:
@GuptaManan100 pointed out that this file is actually generated (see Line 1) using https://github.com/vitessio/vitess/blob/main/test/ci_workflow_gen.go.
I'm not sure what happens to other tests if we change the template that is being used by the `make` target to generate yaml files.
I guess you will need to try and move these changes into the template and if it breaks other tests, then maybe create a new template for mysql80?

Reply:
Yeah, will need to do something there. Running `make generate_ci_workflow` puts `cluster_endtoend_mysql80.yml` back into it's original state where it breaks after the ubuntu-latest image change.

Reply:
Added support here: https://github.com/vitessio/vitess/pull/9370/commits/85494ba94bb67375ce7f810f233f48bc21e88d4b

So now `make generage_ci_workflow` produces what I had previously done in the generated file.